x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -1,14 +1,312 @@
 {
   "template": {
     "mappings": {
+      "dynamic_templates": [
+        {
+          "match_ip": {
+            "match_mapping_type": "string",
+            "match": "ip",
+            "runtime": {
+              "type": "ip"
+            }
+          }
+        },
+        {
+          "message": {
+            "path_match": "message",
+            "mapping": {
+              "type": "match_only_text"
+            }
+          }
+        },
+        {
+          "log_level": {
+            "path_match": "log.level",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
```

Comment:
do we need to specify ignore_above here?

Reply:
Good question. That's what we previously did for all keyword fields and I didn't what to change it.

https://github.com/elastic/elasticsearch/blob/5d6af5890f70cfdb234240f7a1d8c65c6339315d/x-pack/plugin/core/src/main/resources/data-streams-mappings.json#L23-L31

Reply:
In theory no, in practice maybe. Lets assume for whatever reason log.level value is > 32KB, in this case, ES rejects the document and we never want this to happen. The part I don't know is what ignore_malformed does in this scenario, likely worth to test.